src/error/metadata/suggestion.rs

```diff
@@ -240,6 +241,9 @@ UpgradePlan => "Rover has likely reached rate limits while running graph or subg
             }
             CreateOrFindValidPersistedQueryList { graph_id, frontend_url_root } => {
                 format!("Find existing persisted query lists associated with '{graph_id}' or create a new one by heading to {frontend_url_root}/graph/{graph_id}/persisted-queries")
+            },
+            SpecifyRoutingUrl => {
+                String::from("Try specifying a routing URL in the supergraph YAML file. (See https://www.apollographql.com/docs/rover/commands/supergraphs/#yaml-configuration-file)")
```

Comment:
```suggestion
                String::from("Try specifying a routing URL in the supergraph YAML file. See https://www.apollographql.com/docs/rover/commands/supergraphs/#yaml-configuration-file for more details.")
```

Reply:
would running a `subgraph publish` with a `--routing-url` also fix this? should we call that out?